lamp/files/var/www/html/index.html

```diff
@@ -0,0 +1,103 @@
+      <h1 class="title">Please log into your Droplet with SSH to configure the LAMP installation.</h1>
+      <p class="description">See the LAMP One-Click Quickstart guide for detailed assistance.</p>
+      <a class="button" href="https://www.digitalocean.com/docs/one-clicks/lamp/#start">Quickstart Guide</a>
```

Comment:
This link redirects to https://marketplace.digitalocean.com/apps/lamp#getting-started

Reply:
I'll remove the redirect to point to the getting started url.